aikg/flows/chroma_examples.py

```diff
@@ -95,25 +108,28 @@ def chroma_build_examples_flow(
     )
 
     # Create subject documents
-    docs = akio.get_sparql_examples(
-        input_path=chroma_input_dir,
+    docs = get_sparql_examples(
+        dir=chroma_input_dir,
     )
 
     # Vectorize and index documents by batches to reduce overhead
     logger.info(f"Indexing by batches of {chroma_cfg.batch_size} items")
     embed_counter = 0
     for batch in chunked(docs, chroma_cfg.batch_size):
         embed_counter += len(batch)
-        index_batch(batch)
+        for doc in batch:
+            index_batch(doc)
```

Comment:
```suggestion
        index_batch(batch)
```
**question:** is this a mistake? As far as I understand:
* `docs: Iterator[Document]` _contains all documents_
* `batch: list[Document]` _contains `config.batch_size` documents_
* `doc: Document`
But `index_batch()` takes a `list[Document]` as input, not a single `Document`. Perhaps I'm missing something if it works :thinking: . Or does it always index by batches of 1 document?

Reply:
You made me realize of a bug that I fixed in the latest commit. The function `get_sparql_examples()` was returning a list with a document inside instead of a document straight up. Basically `index_batch()` was looping through a list of lists of individual documents. I fixed it now together with your suggestion and everything works 👍